craft_application/commands/base.py

```diff
@@ -117,20 +121,38 @@ def get_managed_cmd(
 
         :param parsed_args: The parsed arguments used.
         :returns: A list of strings ready to be passed into a craft-providers executor.
-        :raises: RuntimeError if this command is not supposed to run managed.
+        :raises: ``RuntimeError`` if this command is not supposed to run managed.
 
         Commands that have additional parameters to pass in managed mode should
         override this method to include those parameters.
+
+        :deprecated: and unused.
```

Comment:
question: If it is deprecated and unused, why do we need to keep it?

Reply:
https://github.com/canonical/craft-application/issues/917